services/common/src/main/java/org/finra/gatekeeper/GatekeeperCommonConfig.java

```diff
@@ -162,6 +167,11 @@ public LdapTemplate ldapTemplate(ContextSource contextSource) {
     @Bean
     public GatekeeperAuthorizationService gatekeeperLDAPAuthorizationService(LdapTemplate ldapTemplate,
```

Comment:
The name of this bean feels dishonest now that we are allowing Headers to be a source of truth for Authorization